Documentation/sourcebuild-in-repos/update-dependencies.md

```diff
@@ -0,0 +1,28 @@
+# Updating a dependency
```

Comment:
This document doesn't discuss all of the issues with picking a version.  Specifically the SBRP scenarios (e.g. the old version came from SBRP but the new version isn't part of SBRP).  Instead of duplicating the version picking topic, what do you think about linking to it in the new-dependencies doc?

Reply:
Added as suggested.